cypress/integration/regions_test/regions_test.js

```diff
@@ -0,0 +1,177 @@
+        cy.get(".input-section").within(() => {
+            cy.get("textarea").clear().type("MAL1:0..30000",{delay:100});
+            cy.get(".radio-group").within(() => {
+                cy.contains("interbase").within(() => {
+                    cy.get(".check").should("not.be.visible");
+                    cy.get(".circle").should("be.visible").click();
+                    cy.wait(500);
+                    cy.get(".check").should("be.visible");
+                })
+            })
+            cy.get("button").filter(':contains("Search")').click();
+        })
+        cy.get(".results").within(() => {
+            cy.get("#region-result-0 > span").should("include.text","MAL1 1..30000");
+        })
+    })
+
+    it("can select to perform strand-specific region search", function(){
+        cy.get(".input-section").within(() => {
+            cy.get("textarea").clear().type("MAL1:0..30000",{delay:100});
+            cy.get(".togglebutton").within(() => {
+                cy.get('.toggle').click();
+                cy.wait(500);
+            })
+            cy.get("button").filter(':contains("Search")').click();
+        })
+        cy.get(".results > #region-result-0").within(() => {
+            cy.get(".icon-arrow-right").should("exist");
+        })
+    })
+
+    it("can extend gene search region by clicking tick marks", function(){
+        cy.get(".extend-region").within(() => {
+            cy.get(".slider-ticks").eq(0).within(() => {
+                cy.contains("100k").click();
+                cy.wait(500);
+            })
+            cy.get(".form-control").eq(1).should("have.value","100k");
+        })
+    })
+
+    it("can extend gene search region by sliders", function(){
+        cy.get(".input-section").within(() => {
+            cy.get("textarea").clear().type("MAL1:0..30000",{delay:100});
+        })
+        cy.get(".extend-region").within(() => {
+            cy.get('input[type="range"]').eq(0)
+            .then($el => $el[0].stepUp(40))
+            .trigger('change');
+            cy.wait(500);
+            cy.get(".form-control").eq(1).should("have.value","10k");
+        })
+    })
+
+    it("can extend gene search region by inputing in the text box", function(){
+        cy.get(".extend-region").within(() => {
+            cy.get("input.form-control").eq(1).clear().type("138k{enter}",{delay:100});
+            cy.get("input.form-control").eq(1).should("have.value","138k");
+        })
+    })
+
+    it("can lock gene search region extension", function(){
+        cy.get(".extend-region").within(() => {
+            cy.get(".icon-lock").click();
+            cy.get(".slider-ticks").eq(0).within(() => {
+                cy.contains("100k").click();
+                cy.wait(500);
+            })
+            cy.get("input.form-control").eq(1).should("have.value","100k");
+            cy.get("input.form-control").eq(2).should("have.value","0");
+        })
+    })
```

Comment:
Can you in addition, run a region search and check that the header for the result is adjusted correctly, e.g. `MAL1:0..30000` becomes `0..130000` for it "can extend gene search region by clicking tick marks."

Beware of overly large regions that could cause the tests to take a long time to complete!
It would also be great if you can use a variety of values for extend regions, like numbers only, numbers with k, numbers with decimal and k. Using 0 as start region for one test is fine, as that tests it shouldn't go negative, but add some searches that don't start at 0 so you can test that the starting location gets extended as well.